greenCan/templates/template.html

```diff
@@ -37,7 +37,7 @@
 					<li class="
 					{% if request.path == reuse_base_url or is_reuse %}
                     current
-                    {% endif %} nav-link"><a href="{{ reuse_base_url }}">Reuse</a></li>
```

Comment:
Instead of changing this, you can change the value of the alias `reuse_base_url` in line 2.

Reply:
Got it, thanks for this